### ISSUE TITLE
[a11y]: Tile [default tile] - SR announcing blank on default tile

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#15266]
### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.41.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO & JAWS

### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.41.1

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO & JAWS

### Description
Screen reader is announcing 'blank' on the default tile

No label is announced for the default tile when navigated through browse mode/ Arrow keys.

### Recommended Fix
Screen reader should announce the content appropriately as shown in the screen. Label should be defined properly.

People who uses screen reader will not be aware what is available and shown in the screen.

### WCAG 2.1 Violation
4.1.2 Name, Role, Value

**Comment by 2nikhiltom** [comment]
Blank announced between "Default tile" & "Link" on v11.84.0 . Ensure that blank is not announced.
